src/test/java/roomescape/integration/ReservationTest.java

```diff
@@ -0,0 +1,235 @@
+package roomescape.integration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.jdbc.core.JdbcTemplate;
+import roomescape.common.BaseTest;
+import roomescape.reservation.controller.response.ReservationResponse;
+import roomescape.theme.domain.Theme;
+import roomescape.theme.service.ThemeRepository;
+
+public class ReservationTest extends BaseTest {
```

Comment:
JUnit5부터는 테스트 클래스 및 메서드를 public으로 선언하지 않아도 됩니다 😄 
```suggestion
class ReservationTest extends BaseTest {
```

Reply:
감사합니다!